docs/table/access_table.rst

```diff
@@ -86,6 +86,7 @@ For all the following examples it is assumed that the table has been created as
 
   >>> from astropy.table import Table, Column
   >>> import numpy as np
+  >>> import astropy.units as u
```

Comment:
While you are at it, could you look at the sentence on line 18 (https://github.com/astropy/astropy/pull/3825/files#diff-40722afe657f5dff3019bd9fd1988234R18) -- it seems mangled.

Also, the print examples are for python2; maybe add parentheses like is done for examples further down?